projects/ion/src/lib/select/select.component.ts

```diff
@@ -76,9 +83,17 @@ export class IonSelectComponent implements OnInit {
     this.search.emit(this.inputValue);
   }
 
+  isInValid(): boolean {
```

Comment:
I believe that is better to use isValid, when we think in affirmative functions it more easy to read.